bitcoin/src/lib.rs

```diff
@@ -83,7 +83,7 @@ extern crate actual_serde as serde;
 #[cfg(test)]
 #[macro_use]
 mod test_macros;
-mod internal_macros;
+pub(crate) mod internal_macros;
```

Comment:
I believe this change is meaningless, or am I missing something. All modules are in scope in the crate they are in, right? What use is a module that cannot be accessed by other code in that crate?

Reply:
Oh I thought it was `pub(crate)` before. Inner modules have access to the items of outer modules.